index.html

```diff
@@ -627,6 +627,70 @@ <h2>DID Documents</h2>
 
     <section>
       <h2>DID Resolvers and DID Resolution</h2>
+
+      <p>
+A <a>DID resolver</a> is a software and/or hardware component that is capable
+of executing the <a>DID resolution</a> and optionally the <a>DID URL dereferencing</a>
+functions for at least one <a>DID method</a>.
+      </p>
+      <p>
+<a>DID resolution</a> is the process of obtaining a <a>DID document</a> for a given <a>DID</a>.
+      </p>
+      <p>
+Building on top of <a>DID resolution</a>, <a>DID URL dereferencing</a> is the process of retrieving a
+representation of a resource for a given <a>DID URL</a>.
+      </p>
+      <p>
+<a>DID resolution</a> and <a>DID URL dereferencing</a> are defined as abstract functions
+(see Section <a href="#resolving-a-did"></a> and Section <a href="#dereferencing-a-did-url"></a>).
+      </p>
+      <p>
+These abstract functions are implemented by <a>DID resolvers</a>. A <a>DID resolver</a> is invoked
```

Comment:
I think the details from here down to line 692 are details that we could be removed from here, and would live better in an auxiliary document to this spec.